data/json/items/tool/explosives.json

```diff
@@ -1615,5 +1615,61 @@
       "explosion": { "power": 69200, "shrapnel": { "casing_mass": 12000, "fragment_mass": 600 } }
     },
     "flags": [ "BOMB", "TRADER_AVOID", "DANGEROUS" ]
+  },
+  {
+    "id": "jug_bomb",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "jug bomb" },
+    "description": "A glass jug with gasoline spilled on the inside and a small pouch of oxidizer powder next to it, which upon mixing will create an incredible shock explosion.  Activating this will mix the two ingredients, after which it's best to throw it as it might go off in your hands!",
```

Comment:
What's a "shock explosion"?

Reply:
Likely a mistranslation or a mix-up of "shock wave" (it's two words in many languages) and "explosion"

Reply:
Not sure what is so incredible about it either.